.azure-pipelines/apiview.yml

```diff
@@ -181,7 +181,7 @@ stages:
                 APIVIEW_COSMOS__CONNECTIONSTRING: $(apiview-cosmos-connectionstring)
                 APIVIEW_GITHUB__CLIENTID: $(apiview-github-clientid)
                 APIVIEW_GITHUB__CLIENTSECRET: $(apiview-github-clientid)
-
+                APIVIEW_AUTHENTICATIONSCHEME: "test"
```

Comment:
curious, what is this for?

Reply:
Test authentication scheme bypasses github authentication in dev environments.

Reply:
and what is the relationship with this and swagger diff?

Reply:
Well, needed so I can get staging to work. Since the test authentication was being triggered for the staging environment

Reply:
It will be better to create a separate PR for that and get it merged independent of this one. 
This will help with:
- Keep commits to one feature only. If we need to cherry-pick or revert a commit, having only one feature makes things easier.
- Ensures features/fixes can be merged independently and possibly faster turn around.
- Code Reviews are easier for the reviewer as everything is under one context

FYI @praveenkuttappan